testing/matrix-sdk-integration-testing/src/tests/reactions.rs

```diff
@@ -34,6 +34,7 @@ use matrix_sdk_ui::timeline::{EventTimelineItem, RoomExt, TimelineItem};
 use crate::helpers::get_client_for_user;
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+#[ignore = "flaky"]
```

Comment:
Can we revert this? I'd rather have one of us investigate this, because it's possible there's a flaw in the SDK code (or that the test could be improved by adding some sleep time, etc.)